app/assets/stylesheets/items.scss

```diff
@@ -3,12 +3,31 @@
 // You can use Sass (SCSS) here: http://sass-lang.com/
 
 .item-data {
-  color: white;
   display: flex;
-  justify-content: space-around;
-  width: 100%;
+  justify-content: space-between;
 
-  &.checked {
+  a {
+    color: white;
+  }
+}
+
+.checked {
+  .item-name {
     text-decoration: line-through;
   }
+}
+
+.item-quantity {
+  display: flex;
+
+  .quantity-text {
+    padding: 0 0.25em;
+  }
+
+  .quantity-button {
+    background-color: white;
+    border-radius: 5px;
+    height: 25px;
```

Comment:
We generally use rem rather than px for except for things such as border: 1px solid black for example. 16px = 1rem. It just works better for responsiveness across breakpoints.